internal/web_api.go

```diff
@@ -70,7 +75,7 @@ func NewAPI(t authorization.TokenGetter, client *http.Client, clientID string, r
 }
 
 // Call sends request to API with given action
-func (a *api) Call(action string, reqPayload interface{}, respPayload interface{}) error {
+func (a *api) Call(action string, reqPayload interface{}, respPayload interface{}, opts ...*CallOptions) error {
```

Comment:
why do you pass opts as varargs? if `CallOptions` isn't some kind of interface then all the options could be provided as single arg

Reply:
I make that because there are only a few methods that require `GET` request so I wanted to make that argument optional. Thanks to that I don't need to change each method with default value (`POST`) and only applied changes to specific methods.